### ISSUE TITLE
Ejercicio 6

### ISSUE THREAD

**Issue by frohe** [c1b3rwall/module_3#6]
Tenemos un archivo wav. Aunque binwalk no lo muestra, haciendo una búsqueda con strings sacamos que hay una cadena 'MThd' que corresponde al header inicial de un archivo midi. Con un editor hexadecimal se extrae de ahí hasta el final del wav pero no parece midi ni se puede escuchar como tal. Con Audacity se importa en bruto y sale el sonido. Dado el contexto, se exporta a wav y se prueba entonces con Sonic Visualiser y aplicando el layer Spectrogram se obtiene la flag.

**Comment by jfzazo** [comment]
Gracias por las instrucciones. Como nota adicional cuando hablas de que exportas el midi a wav y de wav abres con sonic visualizer, he comprobado que desde que se extrae el midi, si se abre con audacity, añades el espectograma, opciones de espectograma (grayscale, min freq 0, max freq 32768) se puede leer directamente.

**Comment by frohe** [comment]
Sí, es que lo vi en el video de estego artístico y son cosas que hago por primera vez. ;)

**Comment by ch4m17ux** [comment]
Excelente guia, sencilla y a lo que es.  Solo me he equivocado de que no sabia en el resultado si era un 1 o una l.

**Comment by rotgogo** [comment]
Muy buena guía. Me faltaba abrirlo con el Sonic Visualiser y aplicar el layer Spectrogram. Muchas gracias frohe!

**Comment by edikuba** [comment]
Quiero agradeceros a todos sus instrucciones, y como a mi personalmente me han hecho falta los pasos intermedios (por falta de conocimientos que he tenido que suplir), quiero aportarlos para aquellas personas alas que les puedan faltar:

1. Abrimos el archivo con Audacity, y no es mucho lo que se puede apreciar.
2. Utilizo:
    `$ strings File_6.wav >> File_6.txt  ↵` # Para analizar el File_6.txt:

    - De la línea 0 a la 620 y no aprecia irregularidades (Otro Magic Numbers o Magic Bytes, Palabras, etc..)
    - Hacemos lo mismo de abajo hacia arriba y en el línea 103629, se aprecia una LIST-INFO Información de metadatos de audio
```
INAM(Nombre (Título)): c1b3rwall
IPRD(Producto (álbum)): c1b3rwall
IART(Artista): c1b3rwall
ICRD(Fecha de creación): 2016
IGNR(Género): hackers 
ISFT(Software): c1b3rwall (libsndfile-1.0.24)
ITRK(Número de pista.): id3 TPE1…
MThd MIDI fmt: Magic Bytes de Archivo de sonido MIDI 
```
Lo que hace evidente la existencia de un archivo incrustado.

3. Antes de intentar otra acción Importamos File_6.wav como datos en bruto: 
    Nota: Otro método sería extraer y guardar a parte el fichero MIDI con Ghex. lo abrimos con Audacity y lo exportamos a 
    .wav e intentar los pasos siguientes..
4. Teniendo en cuenta la información obtenida en el punto 2, abrimos File_6.wav o el vamos al final del archivo y podemos apreciar el añadido... 
     Entonces procedemos a analizar su espectro, experimentando con modificar:
     Frecuencia máxima: 22050
     Ganancia: 16
     Rango: 46
     Escala de grises
     Tamaño de la ventana:
     Tipo de ventana: 2048
     Factor de relleno: Gausiano (a=4.5)
5. Obtenemos la Flag:
